templates/answer/meta.yaml

```diff
@@ -0,0 +1,32 @@
+name: Answer
+description: An open-source knowledge-based community software. You can use it quickly to build Q&A community for your products, customers, teams, and more.
+instructions: Go to /install and select sqlite database
+changeLog:
+  - date: 2022-12-21
+    description: first release
+links:
+  - label: Documentation
+    url: https://answer.dev/docs
+  - label: Github
+    url: https://github.com/answerdev/answer
```

Comment:
You can also add a link to the website: https://answer.dev/

Reply:
fixed